app/src/main/java/fr/free/nrw/commons/di/CommonsApplicationModule.java

```diff
@@ -40,7 +40,7 @@ public Context providesApplicationContext() {
 
     @Provides
     public AccountUtil providesAccountUtil(Context context) {
-        return new AccountUtil(context);
+        return new AccountUtil();
```

Comment:
Feel this is more than a lint issue

Reply:
The private field context in AccountUtil class was an unused field. 
```
private final Context context;

    public AccountUtil(Context context) {
        this.context = context;
    }
```

The constructor here is being used to set the context, but that instance of context isn't really being used. Other methods are static with context being passed as a parameter. I've removed the constructor so it uses the default constructor now.